### PR TITLE
Add Classic Console font

### PR DESCRIPTION
**Pull request by deejayy**
Just a font I use.

### CODE REVIEW

File: src/data/fonts.ts [hunk 1]
```diff
@@ -162,6 +162,14 @@ const rawFonts: Font[] = [
     webPage: 'https://fontesk.com/conta-typeface/',
     srcLink: '/fonts/conta/font.css',
   },
+  {
+    displayName: 'Classic Console',
+    familyName: 'Classic Console',
+    sort: 1,
```
Comment (Gaafar) [inline src/data/fonts.ts:168]: Please change this to 3

File: static/fonts/classic-console/font.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@font-face {
+  font-family: "Classic Console";
+  src: local('*'), url(clacon.woff) format('woff'), url(clacon.ttf) format('truetype');
```
Comment (Gaafar) [inline static/fonts/classic-console/font.css:3]: Can you please include woff2 as it's more optimized, and remove woff & ttf?

File: src/data/fonts.ts (hunk 1 above)
Comment (deejayy) [reply src/data/fonts.ts]: Done!

File: static/fonts/classic-console/font.css (hunk 2 above)
Comment (deejayy) [reply static/fonts/classic-console/font.css]: Done!

**Pull request merged**